jsk_pcl_ros/sample/sample_color_histogram_matcher.launch

```diff
@@ -0,0 +1,44 @@
+<launch>
+  <arg name="gui" default="true" />
+
+  <include file="$(find jsk_pcl_ros)/sample/include/play_rosbag_convenience_store.xml" />
+
+  <node name="color_histogram_publisher"
+        pkg="jsk_pcl_ros" type="color_histogram_publisher.py"
+        output="screen">
+    <rosparam>
+      frame_id: base_link
+      histogram: [
```

Comment:
Please also run `jsk_pcl_ros/color_histogram_visualizer.py` and show this reference histogram in rviz.

Reply:
I think sample launch file should keep as simple as possible.
So, I'd like to reduce the ros node even just a little bit.
But If there are some of the coding rules, I'll follow them.